### PR TITLE
CPT or not-CPT

### PR DESCRIPTION
**Pull request by contactashish13**

### CODE REVIEW

File: includes/admin/models/class-wppr-review-model.php [hunk 1]
```diff
@@ -195,6 +196,23 @@ public function __construct( $review_id = false ) {
...
+		if ( 'wppr_review' === get_post_type( $this->ID ) ) {
+			add_filter( 'wppr_name', array( $this, 'get_name_for_cpt' ), 10, 2 );
+		}
+	}
+
+	/**
+	 * If this is a CPT, use the post title as the product name.
+	 */
+	public function get_name_for_cpt( $name, $id ) {
+		$post   = get_post( $id );
```
Comment (selul) [inline includes/admin/models/class-wppr-review-model.php]: can you use ? https://developer.wordpress.org/reference/functions/get_the_title/

Comment (contactashish13) [reply includes/admin/models/class-wppr-review-model.php]: in get_the_title(), `If the post is protected and the visitor is not an admin, then "Protected" will be displayed before the post title`. Are we fine with this?

Comment (selul) [reply includes/admin/models/class-wppr-review-model.php]: yep, that's fine.

Comment (contactashish13) [reply includes/admin/models/class-wppr-review-model.php]: @selul internally this method does the same thing we are doing so what would be the reason for preferring this?

Comment (contactashish13) [reply includes/admin/models/class-wppr-review-model.php]: @selul done

**Pull request merged**